privatelink/gcp/terraform/terraform.tfvars

```
@@ -0,0 +1,10 @@
+realm           = "your-gcp-project-here"
+region          = "us-central1"
+network_name    = "your-network-name-here"
+subnetwork_name = "your-subnetwork-name-here"
+bootstrap       = "lkc-1n0nvv-6k0qeg.us-central1.gcp.glb.devel.cpdev.cloud:9092"
```

```suggestion
bootstrap       = "lkc-1n0nvv-6k0qeg.us-central1.gcp.glb.devel.cpdev.cloud:9092"
```

@gyang20 we should advertise _public_ names, not private ones... `.confluent.cloud` here.

---

Updated the convention for public use